pkg/collector/corechecks/servicediscovery/apm/detect.go

```diff
@@ -78,25 +93,10 @@ func isInjected(envs map[string]string) bool {
 	return false
 }
 
-const (
-	// ddTraceGoPrefix is the prefix of the dd-trace-go symbols. The symbols we
-	// are looking for are for example
-	// "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer.init". We use a prefix
-	// without the version number instead of a specific symbol name in an
-	// attempt to make it future-proof.
-	ddTraceGoPrefix = "gopkg.in/DataDog/dd-trace-go"
-	// ddTraceGoMaxLength is the maximum length of the dd-trace-go symbols which
-	// we look for. The max length is an optimization in bininspect to avoid
-	// reading unnecesssary symbols.  As of writing, most non-internal symbols
-	// in dd-trace-go are under 100 chars. The tracer.init example above at 51
-	// chars is one of the shortest.
-	ddTraceGoMaxLength = 100
-)
-
 // goDetector detects APM instrumentation for Go binaries by checking for
 // the presence of the dd-trace-go symbols in the ELF. This only works for
-// unstripped binaries.
-func goDetector(pid int, _ []string, _ map[string]string, _ usm.DetectorContextMap) Instrumentation {
+// non stripped binaries.
```

Comment:
It would be better to leave out unrelated changes like this.

Reply:
I restored original comment